packages/gatsby-plugin-sharp/src/duotone.js

```diff
@@ -1,45 +1,44 @@
 const sharp = require(`sharp`)
 
 module.exports = async function duotone(duotone, format, clonedPipeline) {
-  if (duotone) {
-    var duotoneGradient = createDuotoneGradient(
-      hexToRgb(duotone.highlight),
-      hexToRgb(duotone.shadow)
-    )
+  const duotoneGradient = createDuotoneGradient(
+    hexToRgb(duotone.highlight),
+    hexToRgb(duotone.shadow)
+  )
 
-    if (format === `jpg`) {
-      format = `jpeg`
-    }
+  // @todo remove once we upgrade to sharp v0.18 which
+  // adds an alias for "jpg" to toFormat
+  // @see http://sharp.dimens.io/en/stable/changelog/#v0180-30th-may-2017
```

Comment:
Thanks for the heads up! Didn't have the time to look into the changelog for 0.18 yet, merely stumbled upon that note regarding the alias. "attention" is all I had to mention to the people that I showed plugin-sharp to instantly win them over btw. <3